buildSrc/src/main/resources/gocd-docker-server/Dockerfile.server.ftl

```diff
@@ -28,10 +28,18 @@ RUN \
   curl --fail --location --silent --show-error "https://download.gocd.org/binaries/${fullVersion}/generic/go-server-${fullVersion}.zip" > /tmp/go-server-${fullVersion}.zip
 </#if>
 RUN unzip /tmp/go-server-${fullVersion}.zip -d /
-RUN mv /go-server-${goVersion} /go-server && chown -R ${r"${UID}"}:0 /go-server && chmod -R g=u /go-server
```

Comment:
My worry about this is that will silently break if the structure of the zip changes. I tried this locally and the files copied over are 197MB and the files left behind due to this are 17MB. Not sure if the risk is worth that.

Also, there is a LICENSE file there, that would be nice to copy over, if going with this.

Reply:
There are windows, mac, freebsd etc files that are not needed to run on alpine. But, yeah, I guess I'll remove this change if you think that the structure is likely to change.

Reply:
Not sure if it's likely to change, but I'd like something to fail in an obvious way if it changes ...

Reply:
I guess if something important does change, `docker run` will fail to start the server. There used to be some docker tests we were running to make sure basic things(bringing up a server and all agents) worked. I don't see it on build.go.cd anymore though.